tianshou/data/batch.py

```diff
@@ -29,6 +29,43 @@ def _is_batch_set(data: Any) -> bool:
     return False
 
 
+def _is_scalar(value: Any) -> bool:
+    # check if the value is a scalar
+    # 1. python bool object, number object: isinstance(value, Number)
```

Comment:
Does this comment still make sense ?

Reply:
The comment conceptually descibes what is a scalar, it can differ from the implementation details. I think it is ok.